### PR TITLE
feat(app): improve logging, database pool, and auto update shema

### PR DESCRIPTION
**Pull request by mr-ssd**

### CODE REVIEW

File: rootfs/opt/privacyidea/pi-logging.yml [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+    class: logging.StreamHandler
+    level: INFO
+    formatter: detail
+
+loggers:
+  privacyidea:
+    level: INFO
+    handlers: [console]
+
+root:
```
Comment (cparkins) [inline rootfs/opt/privacyidea/pi-logging.yml:18]: @mr-ssd When testing this I found that if this is here logs are duplicated.

Comment (mr-ssd) [reply rootfs/opt/privacyidea/pi-logging.yml:18]: @cparkins thank you for pointing the issue. Could you please add more detail about the log duplication?

Comment (mr-ssd) [reply rootfs/opt/privacyidea/pi-logging.yml:18]: I create new issue #178 for better tracking.

Comment (cparkins) [reply rootfs/opt/privacyidea/pi-logging.yml:18]: @mr-ssd Let me do a build and I can Illustrate the issue.

**Pull request merged**